resources/assets/lib/beatmap-discussions/beatmap-owner-editor.tsx

```diff
@@ -115,7 +115,7 @@ export default class BeatmapOwnerEditor extends React.Component<Props> {
   };
 
   private handleUsernameInput = (e: React.ChangeEvent<HTMLInputElement>) => {
-    this.inputUsername = e.currentTarget.value;
+    this.inputUsername = e.currentTarget.value.trim();
```

Comment:
this prevents entering space at all. The trim should be done in user lookup instead

Reply:
moved